### PR TITLE
Add copyq-clipboard-manager extension

### PR DESCRIPTION
**Pull request by andrewcincotta**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi 👋

Should we close #9139 in favor of this one?

**Comment by andrewcincotta** [comment]
> Hi 👋
> 
> Should we close #9139 in favor of this one?

Sure, my fault.

File: extensions/copyq-clipboard-manager/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CopyQ
+
+Lists CopyQ clipboard entries and allows user to select/paste them.
```
Comment (pernielsentikaer) [inline extensions/copyq-clipboard-manager/README.md:3]: Could you write a bit about CopyQ and a guide for how to install it

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- It's weird that there are more entries in the list than there are items. Could you look into this?

copyq-clipboard-manager 2023-11-15 at 14 28 07

- You can maybe check the default installation paths, [something like here](https://github.com/raycast/extensions/commit/2065b9507ca92d5ad8df480d4d66385cbc0ef16d#diff-1c06f6551afb8db75ccba4044926a86af349f4971d3d27098a5db706ec9b9315R24-R29)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by andrewcincotta** [comment]
@pernielsentikaer I fixed the issue with empty clipboard entries forming. It occurs when the default number of items preference is larger than the number of clipboard entries in the chosen CopyQ tab. Also revised the README to include a description of CopyQ and where to go to install it.

**Comment by pernielsentikaer** [comment]
What about adding the default paths so we could prompt the user as little as possible, not all know how to find the paths, and maybe raise the limit to 25?

**Comment by andrewcincotta** [comment]
> What about adding the default paths so we could prompt the user as little as possible, not all know how to find the paths, and maybe raise the limit to 25?

While I've been waiting for version 1 to be listed on the store I've been overhauling the code to provide a faster display of clipboard contents. In the next version, you can choose a tab before displaying clipboard contents, and all clipboard contents will appear instantly (even on my Mac tab which contains over 1300 items!) 
Can I work on the default path issue in the second version I plan on publishing shortly after?

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**